saleor/product/migrations/0053_product_seo_description.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.0.2 on 2018-03-11 18:54
+import html
+
+from django.core.validators import MaxLengthValidator
+from django.db import migrations, models
+from saleor.core.utils.text import generate_seo_description
+
+
+def to_seo_friendly(cls, text):
```

Comment:
Is `cls` used for something else than getting fields `max_length` param?
Can we pass `max_length` as an `int` instead?

Reply:
I didn't make to allow to be passed as int, the reason behind this was to make the code and reuses more understandable. The idea is to directly take the target field and then directly take the `max_length` attribute. Which would make the code and reuses less repetitive.